draft-ietf-core-new-block.xml

```diff
@@ -14,7 +14,7 @@
      ipr="trust200902">
   <front>
     <title abbrev="Quick Block-Wise Transfer Options">Constrained Application
-    Protocol (CoAP) Block-Wise Transfer Options for Faster
+    Protocol (CoAP) Block-Wise Transfer Options supporting Non-Confirmable
```

Comment:
We are covering both CON and NON. I suggest the folowing: 
```suggestion
    Protocol (CoAP) Block-Wise Transfer Options for Robust 
```

Reply:
OK.  Updated.